lib/editor/selectors/timetable.js

```diff
@@ -72,16 +72,16 @@ export const getTimetableColumns = createSelector(
           if (locations && isLocation) stop = locations.find(l => l.location_id === ps.locationId)
 
           // $FlowFixMe FLEX TODO: improve this to avoid quadruple ternary
-          const stopName = stop ? stop.stop_name : ps.stopId ? ps.stopId : (ps.locationId ? ps.locationId : ps.locationGroupId)
+          const stopName = stop ? stop.stop_name : ps.stopId ? ps.stopId : (ps.locationId ? ps.locationId : ps.areaId)
```

Comment:
Could this work here and at line 84 below?
```suggestion
          const stopName = stop ? stop.stop_name : (ps.stopId || ps.locationId || ps.areaId)
```

Reply:
yep that works! updated in https://github.com/ibi-group/datatools-ui/pull/945/commits/38b445c1b0b6c09d09ad2e67a6b8ecf603632a14